File: tests/java/TestC.java
public class TestC {
    public static void printHelloTestC1() {
        System.out.println("Hello, World!");
        for (int i = 0; i < 5; i++) {
            System.out.println("This is line " + i);
            if (i % 2 == 0) {
                System.out.println("Even number");
            } else {
                System.out.println("Odd number");
            }
        }
        System.out.println("End of printHello");
    }

    public static void printHelloTestC2() {
        System.out.println("Hello, World!");
        for (int i = 0; i < 5; i++) {
            System.out.println("This is line " + i);
            if (i % 2 == 0) {
                System.out.println("Even number");
            } else {
                System.out.println("Odd number");
            }
            System.out.println("Current iteration: " + i);
        }
        System.out.println("End of printHelloAgain");
    }

    public static void main(String[] args) {
        printHello();
        printHelloTestC1();
        printHelloTestC2();
    }
}

File: tests/unknown/nested/testC.rs
fn print_hello_test_c1() {
    println!("Hello, World!");
    for i in 0..5 {
        println!("This is line {}", i);
        if i % 2 == 0 {
            println!("Even number");
        } else {
            println!("Odd number");
        }
    }
}

fn print_hello_test_c2() {
    println!("Hello, World!");
    for i in 0..5 {
        println!("This is line {}", i);
        if i % 2 == 0 {
            println!("Even number");
        } else {
            println!("Odd number");
        }
    }
    println!("End of printHello");
}

fn print_hello_test_c3() {
    println!("Hello, World!");
    for i in 0..5 {
        println!("This is line {}", i);
        if i % 2 == 0 {
            println!("Even number");
        } else {
            println!("Odd number");
        }
    }
}

fn main() {
    print_hello_test_b1();
    print_hello_test_b2();
    print_hello_test_b3();
}